src/Compilers/VisualBasic/Portable/BoundTree/Statement.vb

```diff
@@ -61,6 +61,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Property
     End Class
 
+    Partial Friend Class BoundRaiseEventStatement
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of BoundNode)
```

Comment:
>Protected Overrides ReadOnly Property Children As ImmutableArray(Of BoundNode) [](start = 8, length = 78)

This feels unexpected. I think this property has to be implemented only if the IOperation API corresponding to this bound node is not yet designed or implemented

Reply:
> The reason I had to implement this here is that the InvalidOperation nodes use BoundNode.Children to lazily get the list of children.

Is this a new usage of this API? This feels risky, an additional requirement is added that we didn't have before and there is no guarantee that this new requirement is satisfied by other nodes. I think we should limit the scope of this change to "refactoring", if somethink wasn't lazy before, let's not try to fix this now, but rather open an issue to follow up later. 

---
In reply to: [237313902](https://github.com/dotnet/roslyn/pull/31010#discussion_r237313902) [](ancestors = 237313902,237312399)